### ISSUE TITLE
Lambda won't run: fix permissions problems when zipping

### ISSUE THREAD

**Issue by lusentis** [dawson-org/dawson-cli#1]
If your files are stored with a mode lower than 0644, AWS Lambda won't execute and will report the errors below.

**If you execute the function with the "Test" button from the Lambda console:**

``` json
{
  "errorMessage": "callback called with Error argument, but there was a problem while retrieving one or more of its message, name, and stack"
}
```

**and, in CloudWatch Logs:**
`module initialization error: Error`

The current workaround is to `chmod a+r -Rv .` in your project's root folder.

**Comment by lusentis** [comment]
this seems to happen only when editing files via SMB, with too strict permission umasks. Feel free to re-open if you experience this issue, so we can discuss possible solutions.

**Comment by lusentis** [comment]
it appears that some packages have "broken" permissions (elasticsearch-js for example), and contain `0600` files, which break docker and AWS Lambda.
A fix has been implemented in 3ab1cfac95a2f2dffa57510e9cb9f9df557758c4 which `chmod`s (`chmod -Rf a+rX' the *dist* folder, before creating the zip file or spawning docker). Developers can use the --skip-chmod command line flag to prevent this behaviour (but they must ensure that all file permissions are correct).
